### DISCUSSION TITLE
How to get month in persian with strftime

### DISCUSSION THREAD

**Discussion by mojixcoder** [slashmili/python-jalali#98] [Q&A] [answered]
The output should be something like this 14 (خرداد) 1400.  

Parentheses are just for keeping Khordad between them.

**Comment by slashmili** [comment, marked as answer]
Hi 👋 

I've moved this issue to Discussions

Regarding your question, could you please provide a sample code? 

This section of README might be helpful  https://github.com/slashmili/python-jalali#locale

**Comment by hramezani** [comment]
it seems you can get the Persian month name by using the `locale` when creating the date/datetime object:

```
>>> import jdatetime
>>> fa_datetime = jdatetime.datetime(1397, 4, 23, 11, 40, 30, locale='fa_IR')
>>> fa_datetime.strftime('%b')
'تیر'
>>> fa_date = jdatetime.datetime(1397, 5, 20, locale='fa_IR')
>>> fa_date.strftime('%b')
'مرداد'
```

**Reply by mojixcoder** [reply]
try `{{ date | jformat:"%b" }}` in `django-jalali`. it will give you the eng format. Where do we need to change? here or `django-jalali`?

**Reply by hramezani** [reply]
I think it has to be fixed in `django-jalali`. maybe we can fix it by setting `locale` based on Django [LANGUAGE_CODE](https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-LANGUAGE_CODE) setting

**Reply by mojixcoder** [reply]
Okay, I'll close the PR. Thank you
